pkgs/applications/version-management/yadm/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, git, bash }:
+
+let version = "1.04"; in
+let link = "https://raw.githubusercontent.com/TheLocehiliosan/yadm/${version}"; in
+stdenv.mkDerivation {
+  name = "yadm-${version}";
+  isLibrary = false;
```

Comment:
It's my first time making a derivation, and I may have looked at examples using cabal. If they are not necessary, I will remove them as soon as I get access to a computer.